README.md

```diff
@@ -11,7 +11,7 @@ You can install Codewind locally in VS Code. For more information about installi
 
 Prerequisites
 - Install [VS Code](https://code.visualstudio.com/download).
```

Comment:
ooh, also vs code 1.28 or later, that'll resolve https://github.com/eclipse/codewind/issues/860